tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+
+  // Creating and navigating to new post
+  await page.goto('./wp-admin/post-new.php');
+
+  // On dev instance, need to close modal so test can continue
+  const closeButton = page.locator('.components-modal__header button');
+  if (await closeButton.isVisible()) {
```

Comment:
This is just personal preference, but it seems a bit strange to me to have an `await` statement inside an `if`, maybe we could write this instead:
```
const isCloseButtonVisible = await page.locator('.components-modal__header button').isVisible();
if (isCloseButtonVisible) { ... }
```
What do you think?

Reply:
Actually reading the documentation and seeing other code examples, it is better to have the await in the if statement, seems to validate better. I had it like you suggested but it kept failing.